src/Adapter/Presenter/Product/ProductLazyArray.php

```diff
@@ -466,6 +467,11 @@ public function getFlags()
             );
         }
 
+        Hook::exec('productFlagsModifier', array(
```

Comment:
is the name of the hook consistent with the others?

Reply:
i was wondering how to name this hook, we could have:
`actionProductFlagsModifier`

but it's not really any action

`displayProductFlagsModifier`

is more likely using to display something, not alter

so i decided to do this without prefix as some others hooks with same purpose, it should be it or `action` prefix, i am open for suggestions here

Reply:
As far as I can tell, `*Modifier` hooks start with `action`. :)

Reply:
Let's go for `action` prefix, thanks @Quetzacoalt91 !

Reply:
@mickaelandrieu done, do i need to squash those two commits or it is no longer needed in current workflow?